pandas/tests/scalar/test_timedelta.py

```diff
@@ -676,9 +562,6 @@ def test_timedelta_hash_equality(self):
         d = {td: 2}
         assert d[v] == 2
 
-        tds = timedelta_range('1 second', periods=20)
```

Comment:
why did u move this? and not all

Reply:
> why did u move this? 

timedelta_range --> not scalar

> and not all

You stopped in the middle of a